mx.truffleruby/mx_truffleruby.py

```diff
@@ -109,10 +109,24 @@ def ruby_testdownstream_hello(args):
     build_truffleruby()
     mx.run(['bin/truffleruby', '-e', 'puts "Hello Ruby!"'])
 
+def ruby_testdownstream_aot(args):
+    if len(args) != 1:
+        mx.abort("Missing path to AOT image: mx ruby_testdownstream_aot <aot_bin>")
+
+    mx.run([args[0], 'spec/mspec/bin/mspec', 'run',
+           '--config', 'spec/truffle.mspec',
+           '--excl-tag', 'fails',
+           '--excl-tag', 'graalvm',
+           '--excl-tag', 'aot',
+           '-t', args[0],
+           '-T-XX:YoungGenerationSize=2G', '-T-XX:OldGenerationSize=4G',
+           '-T-Xhome=' + os.getcwd(),
+           ':language'])
```

Comment:
Could you reuse the `ruby_run_specs` method above and adapt it to pass a launcher (the AOT binary for this) to run it?